app/builders/template_builder/from_compose.rb

```diff
@@ -1,35 +1,35 @@
 module TemplateBuilder
-  class FromFig
+  class FromCompose
 
-    attr_reader :fig_yml, :options
+    attr_reader :compose_yaml, :options
 
     def initialize(options)
-      @fig_yml = options.delete('fig_yml')
+      @compose_yaml = options.delete('compose_yaml')
       @options = options
     end
 
     def create_template(persisted=true)
       Template.new(self.options) do |template|
-        template.images = images_from_fig if fig_yml
+        template.images = images_from_compose if compose_yaml
         template.save if persisted
       end
     end
 
     private
 
-    def images_from_fig
-      fig_services = YAML.safe_load(fig_yml) || {}
-      fig_services.map { |name, service_def| image_from_fig_service(name, service_def) }
+    def images_from_compose
+      compose_services = YAML.safe_load(compose_yaml) || {}
+      compose_services.map { |name, service_def| image_from_compose_service(name, service_def) }
     end
 
-    def image_from_fig_service(name, service_def)
+    def image_from_compose_service(name, service_def)
```

Comment:
Assignment Branch Condition size for image_from_compose_service is too high. [21/15]